api/base/serializers.py

```diff
@@ -1718,6 +1723,28 @@ def create(self, validated_data):
         return self.make_instance_obj(collection)
 
 
+class LinkedPreprintsRelationshipSerializer(LinkedNodesRelationshipSerializer):
+    data = ser.ListField(child=LinkedPreprint())
+
+    def get_self_url(self, obj):
+        return obj['self'].linked_preprints_self_url
+
+    def get_related_url(self, obj):
+        return obj['self'].linked_preprints_related_url
+
+    class Meta:
+        type_ = 'linked_preprints'
+
+    def make_instance_obj(self, obj):
```

Comment:
Not a blocker and probably out of scope for this PR, but would be nice if `make_instance_obj` took a `type` argument and didn't need to be redefined in each class.

Reply:
Makes sense but I didn't do that here, because the deleted fields are different on nodes and preprints so I just redefined.